lib/vsc/utils/fancylogger.py

```diff
@@ -174,6 +174,10 @@ class FancyLogger(logging.getLoggerClass()):
     # this attribute can be checked to know if the logger is thread aware
     _thread_aware = True
 
+    # default class for raiseException method, that can be redefined by deriving loggers
+    DEFAULT_RAISE_EXCEPTION_CLASS = Exception
+    DEFAULT_RAISE_EXCEPTION_LOG_METHOD = lambda c, msg: c.warning(msg)
```

Comment:
I wouldnt call it default if it's supposed to be changed by something later on.
so just go with `RAISE_EXCEPTION_CLASS`